tests/queries_tests/task_test.js

```diff
@@ -78,4 +101,10 @@ describe('task', () => {
     })
   )
 
+  it('creates tasks for a user from a list of template tasks', () =>
```

Comment:
Can you check the `due_date` of the task created?  In fact can we check all the attributes of the task created?